src/groups/mwc/mwcc/mwcc_orderedhashmap.t.cpp

```diff
@@ -853,15 +853,16 @@ static void test14_localIterator()
     // Add keys such that they all map to same bucket in the table, while
     // ensuring that table is not rehashed.
     for (unsigned int i = 0; i < (bucketCount - 2); ++i) {
-        key += bucketCount;
+        key += static_cast<int>(bucketCount);
```

Comment:
And if we do the above, we don't need any casts here or below!  (Though we probably would need to change the `unsigned int` to `size_t`.)